blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/preload/FavoritesStatePreloader.java

```diff
@@ -56,7 +57,7 @@ protected FetchData getFetchData(@Nonnull BlueUrlTokenizer blueUrl) {
         User jenkinsUser = User.current();
 
         if (jenkinsUser != null) {
-            UserImpl blueUser = new UserImpl(jenkinsUser);
+            UserImpl blueUser = new UserImpl(OrganizationUtil.getOrganization(OrganizationUtil.getOrganizationNameFromURL(), true), jenkinsUser);
```

Comment:
@i386 if the organization is in a folder that won't work. In fact the whole point of this PR is to support permissions when the organization is not in Jenkins root.

The reason why I think that in the preloaders we should get it from the url is because the whole API supports multiple organizations, given the OrgFactory with the containingOrg and the of methods.

If we don't take into account which org the user is accessing through the url the only solution we have left is accessing to the one on Jenkins root (which in my case the is none) or get the first one, which makes pointless supporting organizations in folders.

I agree with you that we had to remove every place where the url was accessed to get the organization and the Jenkins.getInstance() use, but the case of the preloaders is different because is where the front-end get the information about the org the user is accessing.
WDYT?